docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -349,6 +349,8 @@ To create a credit class, run the following command:
 regen tx ecocredit create-class [issuers] [credit-type-abbrev] [metadata]
 ```
 
+The `credit-type-abbrev` must be one of the allowed credit types for your chain.
+By default, the only allowed credit type is `C`, so unless you've manually configured other credit types in your chains genesis file, you will need to use `C` here.
```

Comment:
I think we should add a section to the allowlist and class fee section rather than instructing users to use carbon. We can link to the docs and demonstrate how to look up the credit types as we do for the allowlist and class fee.

Reply:
That makes sense, if you share some of the references I'd be glad to give it a go modifying the tutorial as you suggest. Won't be able to do that until next week, but happy to do it. I'm just in need of some references/further info to better understand what you'd like to do. But if you'd just prefer to do it, I'm fine with that too